TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriveTrain.java

```diff
@@ -0,0 +1,246 @@
+package org.firstinspires.ftc.teamcode;
+
+import android.util.Log;
+
+//import com.qualcomm.hardware.bosch.BNO055IMU;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.Gamepad;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation;
+
+public class DriveTrain {
+    String DRIVE_TRAIN_CAPTION = "Drive Status";
+    Telemetry telemetry;
+    HardwareInnov8Robot robot;
```

Comment:
rename this variable

Reply:
I will change the name when we come up with a theme